core/src/main/kotlin/di/Module.kt

```diff
@@ -91,10 +91,9 @@ import org.koin.dsl.module
 
 /**
  * Creates the Koin module for the ORT server. The [config] is used to configure the application and the database. For
- * integration tests, the [setupDatabase] flag can be set to `false` as the data source will be provided by the
- * testcontainer there.
+ * integration tests, the [database][db] from the testcontainer can be provided directly.
  */
-fun ortServerModule(config: ApplicationConfig, setupDatabase: Boolean = true) = module {
+fun ortServerModule(config: ApplicationConfig, db: Database?) = module {
```

Comment:
Should `db` have a default value of `null` as `setupDatabase` also had a default value before?

Reply:
I left it out because there is only one caller which always provides a value for that argument.